src/js/get_profile_data.js

```diff
@@ -196,9 +196,9 @@
   } else {
     // Scrape alias data from Profile page (Local)
 
-    await browser.runtime.sendMessage({
-      method: "displayBrowserActionBadge",
-    });
+    // await browser.runtime.sendMessage({
+    //   method: "displayBrowserActionBadge",
+    // });
```

Comment:
No badge for premium release?

Reply:
Only if we had a banner/something to mention about it. There's no specific "announcement" panel.